perfmetrics/scripts/continuous_test/gcp_ubuntu/e2e_tests/build.sh

```diff
@@ -34,4 +35,4 @@ git checkout $commitId
 
 echo "Running e2e tests on installed package...."
 # $1 argument is refering to value of testInstalledPackage
-./tools/integration_tests/run_e2e_tests.sh $RUN_E2E_TESTS_ON_INSTALLED_PACKAGE $SKIP_NON_ESSENTIAL_TESTS_ON_PACKAGE $BUCKET_LOCATION $RUN_TEST_ON_TPC_ENDPOINT
+./tools/integration_tests/run_e2e_tests.sh $RUN_E2E_TESTS_ON_INSTALLED_PACKAGE $SKIP_NON_ESSENTIAL_TESTS_ON_PACKAGE $BUCKET_LOCATION $RUN_TEST_ON_TPC_ENDPOINT $RUN_TESTS_WITH_PRESUBMIT_FLAG
```

Comment:
what's the difference between SKIP_NON_ESSENTIAL_TESTS_ON_PACKAGE and the new flag being introduced ?

Reply:
SKIP_NON_ESSENTIAL_TESTS_ON_PACKAGE does not differentiate between presubmit vs periodic runs, while the new flag does. SKIP_NON_ESSENTIAL_TESTS_ON_PACKAGE is set to `true` for both presubmit and periodic runs, so it wasn't useful for me to disable the experimental-feature test in question here specifically for presubmit runs.